acceptance/pre_suites/41_copy_mock_pe_tarball.rb

```diff
@@ -0,0 +1,5 @@
+test_name "copy mock tarball PE tarball" do
```

Comment:
Looks like this is copying the fixtures dir, but the name implies it only copied the tarball...  There is/can be more in fixtures... should probably change the name of the test (and the file) to match what it is doing.